src/main/resources/com/google/api/codegen/copyright-google.txt

```diff
@@ -1 +1 @@
-Copyright 2017 Google LLC
+Copyright 2018 Google LLC
```

Comment:
@lukesneeringer why did the comma disappear after the 2018 in the last update of the copyright line?

Reply:
@garrettjonesgoogle See https://opensource.google.com/docs/releasing/preparing/, the section "Apache header".

Reply:
ugh... fine...

Reply:
The comma should not be there. :-)